ome_zarr/writer.py

```diff
@@ -432,8 +435,85 @@ def write_image(
         axes=axes,
         coordinate_transformations=coordinate_transformations,
         storage_options=storage_options,
+        **metadata,
+    )
+
+
+def write_multiscale_image_labels(
```

Comment:
I wouldn't do that, because I think we should mirror the functionality for normal multiscale data and have
- `write_multiscale_image_labels`, corresponding to `write_multiscale` (which I have implemented here already)
- `write_image_label`, corresponding to `write_image` (not implemented yet, since I wanted to wait till we agree on the signature for `write_multiscale_image_labels`; this would use `write_multiscale_image_labels` internally).

If you want to simplify the names we can rename `write_multiscale_image_labels` -> `write_multiscale_labels` and `write_image_labels` -> `write_labels` though; that would be fine with me.

Reply:
Understood. For me, the most confusing part is that a label image must implement the multiscales specification so the `_multiscales` bit feels redundant.
That being said, there is already a (confusing) distinction between `write_image` and `write_multiscales` in the current API.

Reply:
> Understood. For me, the most confusing part is that a label image must implement the multiscales specification so the `_multiscales` bit feels redundant.

But we need to keep the `_multiscales` to keep the distinction between passing a single array or the MIP. 

> That being said, there is already a (confusing) distinction between `write_image` and `write_multiscales` in the current API.

Yes, that's why we need the `_multiscales` here.

(I will go ahead and simplify a bit by removing the "image" as layed out above.)